packages/discovery-provider/src/tasks/update_aggregates.py

```diff
@@ -82,28 +82,45 @@
   group by
     repost_item_id
 ),
+track_comments as (
+  select
+    entity_id as comment_entity_id,
+    count(*) as comment_count
+  from
+    comments c
+  where
+    c.is_delete is false
+    and c.is_visible is true 
+    and c.entity_type = 'Track'
+  group by
+    comment_entity_id
+),
 new_aggregate_track as (
   select
     ap.track_id,
     coalesce(ps.save_count, 0) as save_count,
     coalesce(pr.repost_count, 0) as repost_count
+    coalesce(pc.comment_count, 0) as _count
```

Comment:
Should this be `comment_count`?

Reply:
ah nice, i need to remove this line actuallyi